addons/mail/static/src/model/model_manager.js

```diff
@@ -34,19 +34,29 @@ class ModelManager {
          * The messaging env.
          */
         this.env = env;
+
+        //----------------------------------------------------------------------
+        // Various variables that are necessary to handle an update cycle. The
+        // goal of having an update cycle is to delay the execution of computes,
+        // life-cycle hooks and potential UI re-renders until the last possible
+        // moment, for performance reasons.
+        //----------------------------------------------------------------------
+
+        /**
+         * Set of records that have been created during the current update
+         * cycle. Useful to trigger _created.
+         */
+        this._createdRecords = new Set();
         /**
-         * Whether this is currently handling an "update after" on a record.
-         * Useful to determine if we should process computed/related fields.
+         * Whether any change happened during the current update cycle. Useful
+         * to know if rev number has to be incremented.
          */
-        this._isHandlingToUpdateAfters = false;
+        this._hasAnyChangeDuringCycle = false;
```

Comment:
Poorly worded comment.

For the _"what"_ (1st sentence), it should be more impactful: _"This attribute tracks whether something has changed during the current update cycle."_ (it means the same, but I get it on my 1st reading).

For the _"why"_ (2nd sentence), it's obscure because it doesn't explain how incrementing rev number does something relevant (it's the 1st time in file rev number is mentioned). I think it's best not to mention rev number and give the exact intent, which is to notify components of changes.